modules/rosa-sts-cluster-terraform-destroy.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly-terraform.adoc
+//
+:_content-type: PROCEDURE
+
+[id="sd-terraform-cluster-destroy_{context}"]
+= Deleting your ROSA cluster with Terraform
+
+Terraform uses the `destroy` command to remove all resources that you created with the `terraform apply` command.
+
+[NOTE]
+====
+You should not modify your Terraform .tf files nor the "terraform.tfvars" file before destroying your resources. Terraform uses these variables to select the correct resources to delete.
+====
+
+.Procedure
+. In the directory where you ran `terraform apply` to create your cluster, run the following command to delete the cluster:
++
+[source,terminal]
+----
+$ terraform destroy
+----
++
+Enter the name of the cluster you want to delete. You see the destruction plan, and you are prompted to enter `yes` to delete your cluster. This process cannot be canceled. After the `destroy` command has completed, your account and Operator roles as well as your cluster are deleted.
```

Comment:
Not sure we should use "see" here?

Reply:
Would a screenshot help here?

Reply:
This isn't a bad idea. Let me experiment with this.